src/main/java/uk/ac/cam/cl/dtg/isaac/dto/IsaacNumericQuestionDTO.java

```diff
@@ -36,9 +36,8 @@
 @ValidatesWith(IsaacNumericValidator.class)
 public class IsaacNumericQuestionDTO extends IsaacQuestionBaseDTO {
     private Boolean requireUnits;
-    private Integer significantFiguresMin;
-    private Integer significantFiguresMax;
```

Comment:
Why can you delete these here? Are sig-figs only used in the API?

Reply:
Yes, and more importantly they were `@JsonIgnore`-d to ensure they never got sent to the frontend since we do not want them to be there! I decided to remove them to check it didn't break anything, and since it didn't I left it in.